tests/ResourceFileCreateCommandTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: alex
+ * Date: 18/03/19
+ * Time: 5:31 AM
+ */
+
+namespace CrestApps\CodeGenerator\Tests;
+
+use Illuminate\Support\Facades\Artisan;
+use Illuminate\Support\Facades\File;
+
+class ResourceFileCreateCommandTest extends TestCase
+{
```

Comment:
Two tests with my original bug case minimised as far as I could get it - both to verify that I correctly hooked up the artisan commands (to enable more complete testing later on) and to ensure what I aimed to fix actually gets, and stays, fixed.